source/zh-cn/docs/gitlab-pages.md

```diff
@@ -1,11 +1,14 @@
 ---
-title: GitLab Pages
+title: 将 Hexo 部署到 GitLab Pages
 ---
 
-1. Create a new repository named <b>*username*.gitlab.io</b>, where username is your username on GitLab. If you have already uploaded to other repo, rename the repo instead.
-2. Enable Shared Runners via `Settings -> CI / CD -> Shared Runners`.
-3. Push the files of your Hexo folder to the repository. The `public/` folder is not (and should not be) uploaded by default, make sure the `.gitignore` file contains `public/` line. The folder structure should be roughly similar to [this repo](https://gitlab.com/pages/hexo).
-4. Add `.gitlab-ci.yml` file to your repo (alongside _config.yml & package.json) with the following content:
+在本教程中，我们将会使用 GitLab CI 将 Hexo 博客部署到 GitLab Pages 上。
+
+1. 新建一个 repository。如果你希望你的站点能通过 `<你的 GitLab 用户名>.gitlab.io` 域名访问，你的 repository 应该直接命名为 `<你的 GitLab 用户名>.gitlab.io`。
+2. 如果你使用的是 Self-Host 的 GitLab，你需要在 `Settings -> CI / CD -> Shared Runners` 启用 GitLab CI。
```

Comment:
"Shared Runners" is available on Gitlab.com, not just self-host. It's enabled by default, so the step is just making sure.

Reply:
Agree, "self-host" may make people think it refers to some other settings.

Reply:
@SukkaW perhaps you could add a note at the end of this step saying this is usually enabled by default.

Reply:
Notes updated.